migration.md

```diff
@@ -68,14 +68,14 @@ The procedure for migrating the files and the database works like this:
     1. **Please note:** Unfortunately it is not possible to import a database dump from a former database owner with the name `nextcloud`. You can check if that is the case with this command: `grep "Name: oc_appconfig; Type: TABLE; Schema: public; Owner:" database-dump.sql | grep -oP 'Owner:.*$' | sed 's|Owner:||;s| ||g'`. If it returns `nextcloud`, you need to rename the owner in the dump file manually. A command like the following should work, however please note that it is possible that it will overwrite wrong lines. You can thus first check which lines it will change with `grep "Owner: nextcloud$" database-dump.sql`. If only correct  looking lines get returned, feel free to change them with `sed -i 's|Owner: nextcloud$|Owner: ncadmin|' database-dump.sql`. 
 1. Next, copy the database dump into the correct place and prepare the database container which will import from the database dump automatically the next container start: 
     ```
-    sudo rm /var/lib/docker/volumes/nextcloud_aio_database_dump/_data/database-dump.sql
-    sudo cp database-dump.sql /var/lib/docker/volumes/nextcloud_aio_database_dump/_data/
-    sudo chmod 777 /var/lib/docker/volumes/nextcloud_aio_database_dump/_data/database-dump.sql
-    sudo rm /var/lib/docker/volumes/nextcloud_aio_database_dump/_data/initial-cleanup-done
+    sudo docker run --rm --volume nextcloud_aio_database_dump:/mnt/data:rw alpine rm /mnt/data/database-dump.sql
+    sudo docker cp database-dump.sql nextcloud-aio-database:/mnt/data/
```

Comment:
do relative paths work?

Reply:
Yes, it does. Even relative paths inside docker work but then you have to rely on [Dockerfile WORKDIR](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#workdir).

Reply:
Cool! so can we maybe also use this in the other docker cp cases or is this not a good idea?

What I mean is e.g. changing ``sudo docker cp --follow-links /path/to/nextcloud/skeleton/ nextcloud-aio-nextcloud:/mnt/ncdata/skeleton/`` to ``sudo docker cp --follow-links ./skeleton/ nextcloud-aio-nextcloud:/mnt/ncdata/skeleton/``

Reply:
I think relative paths (just filenames) for files and absolute for directories are fine. This makes it easier to see the difference and users intuitively see that they have to change `/path/to/nextcloud/skeleton/`.

Reply:
Convinced 👍